lib/teleterm/config.go

```diff
@@ -54,5 +55,11 @@ func (c *Config) CheckAndSetDefaults() error {
 		return trace.BadParameter("only unix sockets are supported")
 	}
 
+	if len(c.ShutdownSignals) == 0 {
+		// If ShutdownSignals is empty, the service will be immediately shut down on start as
+		// Signal.Notify relays all signals if it's given no specific signals to watch for.
+		c.ShutdownSignals = []os.Signal{os.Interrupt, syscall.SIGTERM}
```

Comment:
Does this have to build on windows as well? We don't have a build constraint for `unix` or `!windows`

Reply:
edit: nevermind, looks like Go will happily pass along a `SIGTERM` instead of a handful of equivalent-ish windows signals

Reply:
If I recall correctly, `syscall.SIGTERM` with `signal.Notify` on Windows does actually work and notifies on some windows specific events e.g shut down or logging off.